joint_limits:
  joint_l_f:
    has_position_limits: false
    has_velocity_limits: false
    max_velocity:  1.0
    has_acceleration_limits: false
    has_jerk_limits: false
    has_effort_limits: false
  joint_l_b:
    has_position_limits: false
    has_velocity_limits: false
    max_velocity: 1.0
    has_acceleration_limits: false
    has_jerk_limits: false
    has_effort_limits: false
  joint_r_f:
    has_position_limits: false
    has_velocity_limits: false
    max_velocity: 1.0
    has_acceleration_limits: false
    has_jerk_limits: false
    has_effort_limits: false
  joint_r_b:
    has_position_limits: false
    has_velocity_limits: false
    max_velocity: 1.0
    has_acceleration_limits: false
    has_jerk_limits: false
    has_effort_limits: false
